ncl/ncl_entries/sin_trigonometry.ipynb

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
<p>Might not be relevant with other comments, but I'd personally recommend numpy's sine function over the math library, especially since it can handle arrays</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Yeah, this is a personal take, but I think some of the relevant content for trig functions in Python that I'd want to know is more along the lines of where the find the functions and what ones to use when (probably numpy ones most often for performance reasons and compatibility w/ xarray, dask, and the "Pangeo" stack).  

Again this is totally a design thing.